etc/kayobe/ansible/lab-hosts.yml

```diff
@@ -0,0 +1,28 @@
+---
+- hosts: all
```

Comment:
nit: it would be a little simpler to target localhost and remove the run_once

Reply:
good thinking